packages/desktop-client/src/components/util/AmountInput.tsx

```diff
@@ -70,7 +70,9 @@ export function AmountInput({
 
   function getAmount(negate) {
     const valueOrInitial = Math.abs(
-      amountToInteger(evalArithmetic(value, initialValueAbsolute)),
+      amountToInteger(
+        evalArithmetic(value, stringToInteger(initialValueAbsolute)),
+      ),
```

Comment:
I'm not entirely confident in this change. Would appreciate a second pair of eyes. cc @joel-jeremy 

This is necessary because the 2nd arg of `evalArithmetic` should always be a `number`. However, it is a `string` here.

Reply:
I think we can use `initialValue` here as the default.